templates/data/relational-databases.html

```diff
@@ -0,0 +1,292 @@
+{% extends 'base_index.html' %}
+
+{% block title %}Relational databases within Canonical{% endblock %}
+
+{% block meta_description %}Enterprise-grade database offering with built-in high availability, backup, monitoring and automated security updates. It is ideal for businesses that need a reliable database to power their mission-critical applications.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1r1xnNihaTphO_eikifUEO7ocYfTDnbuRIaIfUCmya1c/edit{% endblock meta_copydoc %}
+
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block content %}
+
+<section class="p-strip is-shallow">
+  <div class="row--25-75 is-split-on-medium">
+		<div class="col">
+			<div class="p-section--shallow">
+					<h1 class="u-no-margin--bottom">What is a relational database</h1>
+					<h2>When to use them and how Canonical services can make your relational database operations secure and efficient.</h2>
+			</div>
+			<hr class="u-hide--small">
```

Comment:
```suggestion
			<hr>
```

Reply:
you told me to hide it in small screens in a previous comment....